src/tic-tac-toe.ts

```diff
@@ -0,0 +1,24 @@
+type Square = string;
+type Row = Square[];
+type Board = Row[];
+
+class Game {
+  public board: Board;
+  private currentPlayer: string;
+  constructor() {
+    this.currentPlayer = "X";
```

Comment:
I would try to make "X" and "Y" an Enum called `PlayerPiece` or something like that. Then you could initialize the board with `undefined` instead of empty strings (which you could still do now).

Reply:
Done!